app/controllers/health_controller.rb

```diff
@@ -0,0 +1,13 @@
+class HealthController < ApplicationController
+  def workers
+    status = worker_health_checker.all_healthy? ? :ok : :internal_error
+
+    render json: worker_health_checker.statuses, status: status
```

Comment:
hmm. is there now a race condition? since `.statuses` are not cached afaik, something could finish and go healthy in between the `.all_healthy?` call and then subsequent call to `.statuses` ?

maybe not to worry about if this is looking for catastrophic failures.